scripts/tag-promotion.sh

```diff
@@ -49,9 +49,9 @@ case "${choice}" in
 esac
 
 REMOTE="git@github.com:GoogleContainerTools/kpt-config-sync.git"
-# Fetch all existing tags
-git fetch "${REMOTE}" --tags > /dev/null
-echo "+++ Successfully fetched tags"
+# Fetch all existing tags/heads
+git fetch "${REMOTE}" +refs/heads/*:refs/remotes/upstream/* --tags > /dev/null
```

Comment:
What if the remote name is not set to `upstream`?

Can we instead using `git fetch --all`?

Reply:
This fetch will create a local namespace named `upstream` even if the user has not already defined a remote for it yet.

`git fetch --all` assumes the user already defined a remote for the GoogleContainerTools repo. Which is common and they probably should, but I'm trying to minimize assumptions for this script